README.md

```diff
@@ -1 +1,30 @@
-# personal-portfolio
\ No newline at end of file
+Portfolio: setup and mobile first
+This was created for microverse's activity and also I created the repository "personal-portfolio"
+Built With
+HTML y CSS
+Git, Github and Visual Studio Code
+Linters
+Getting Started
+To get a local copy up and running follow these simple example steps.
+
+Choose a directory
+Open a Terminal
+git clone https://github.com/elad237/personal-portfolio.git
+go to directory ""development 
+npm install
+Authors
+👤 tumbu
+
+GitHub: @elad237
+Twitter: @Elad59380989
+LinkedIn: tumbu
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [OPTIONAL] I recommend to adjust Main header of Your  README file Just Like [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) .